hw20/group_vars/ubuntu/ubuntu.yml

```diff
@@ -0,0 +1,11 @@
+---
+#Ubuntu vars
+group_hw20: ZONE_1
+sudo_groups: sudo,docker
+user_name_hw22: "hw21_1"
+hw21_user:
+ - { name: hw21_1, groups: "sudo, docker", shell: /bin/sh, home: /opt/hw21 }
+curl_files:
+ - { src: "curl.deb", dest: "/tmp/curl.deb", owner: "hw20", mode: "0744" }
+hw20_user:
+ - { name: hw20, shell: /bin/sh, home: /opt/hw20, groups: "sudo, docker" }
```

Comment:
better to keep the same order for parameters

Reply:
{ name: hw21_1, groups: "sudo, docker", shell: /bin/sh, home: /opt/hw21 }
{ name: hw20, shell: /bin/sh, home: /opt/hw20, groups: "sudo, docker" }

will be easier to read

Reply:
fix